docs/JestObjectAPI.md

```diff
@@ -500,14 +506,16 @@ Example test:
 ```js
 const video = require('./video');
 
+afterEach(() => {
+  jest.restoreAllMocks();
```

Comment:
Perhaps instead of a tip, short comment could be added here explaining that this will restore state of the spy?

Reply:
Hi @mrazauskas, I added a comment on afterEach hook on this commit https://github.com/facebook/jest/pull/13000/commits/6fa35fe8084565f7d6cf973c8a1c6970c13da702